faiss-search/README.md

```diff
@@ -93,22 +93,22 @@ metas:
   prefetch: 10
```

Comment:
If possible, let's update also `README.md` to match actual files

Reply:
updated